app/assets/javascripts/controllers/live_metrics/util/metrics-utils-factory.js

```diff
@@ -2,6 +2,16 @@ angular.module('miq.util').factory('metricsUtilsFactory', function() {
   return function (dash) {
     var UNKNOWN_ERROR_STR = __('Something is wrong, try reloading the page');
 
+    function inplaceCalcDataDifferentials(data) {
+      data.forEach(function(v, i) {
+        if ((data[i] !== null) && (data[i + 1] !== null)) {
+          data[i] = data[i + 1] - data[i];
```

Comment:
@yaacov I am not sure if we want to do this inplace... wouldn't this function always result in an array where the last item is `null`?